ArticleTemplates/assets/js/modules/ads.js

```diff
@@ -101,43 +101,39 @@ define([
                 });
             },
             getBannerPosCallback : function(callbackNamespace, callbackFunction) {
```

Comment:
These were pre-existing parameters I just amended the comment above. However lets remove these and see if everything still works as expected on Android, as it should be ok if not being used.